paper/src/main/java/com/denizenscript/denizen/paper/events/PlayerLinksSendScriptEvent.java

```diff
@@ -0,0 +1,54 @@
+package com.denizenscript.denizen.paper.events;
+
+import com.denizenscript.denizen.events.BukkitScriptEvent;
+import com.denizenscript.denizen.utilities.Utilities;
+import com.denizenscript.denizen.utilities.implementation.BukkitScriptEntryData;
+import com.denizenscript.denizencore.objects.core.ListTag;
+import com.denizenscript.denizencore.scripts.ScriptEntryData;
+import io.papermc.paper.connection.PlayerGameConnection;
+import org.bukkit.entity.Player;
+import org.bukkit.event.EventHandler;
+import org.bukkit.event.Listener;
+import org.bukkit.event.player.PlayerLinksSendEvent;
+
+public class PlayerLinksSendScriptEvent extends BukkitScriptEvent implements Listener {
+
+    // <--[event]
+    // @Events
+    // player links send
+    //
+    // @Group Player
+    //
+    // @Plugin Paper
+    //
+    // @Triggers when the list of links is sent to the player
+    //
+    // @Determine
+    // "LINKS:<ListTag(MapTag)>" to change what links should player see. Each map needs to have "uri" key which represents URI address of this link and either display or a type. Valid types are listed at <@link url https://hub.spigotmc.org/javadocs/spigot/org/bukkit/ServerLinks.Type.html>
+    //
+    // @Player when the event part of player game connection

```

Comment:
I feel like this isn't super descriptive to the average user? might be a good idea to have a full on `@Warning this can fire in early player login, before player is in world` (can discuss wording on Discord) & reference to that in `@Player`?